src/index.html

```diff
@@ -7,12 +7,61 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="container"
+      data-qa="card"
+    >
+      <img
+        class="container__image"
+        src="images/imac.jpeg"
+        alt="Apple Device(imac)"
+      />
+      <div class="container__product-info">
+        <h3 class="product-info-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="product-info-text">Product code: 195434</p>
+      </div>
+      <div class="container__stars-reviews">
+        <div class="container__stars-reviews-star">
+          <div class="container__stars-reviews-star-icon"></div>
+          <div class="container__stars-reviews-star-icon"></div>
+          <div class="container__stars-reviews-star-icon"></div>
+          <div class="container__stars-reviews-star-icon"></div>
+          <div class="container__stars-reviews-star-icon"></div>
+        </div>
```

Comment:
You need to use this block from the previous task and add the modifier class
```suggestion
        <div class="star stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```